<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parser</class>
 <widget class="QWidget" name="Parser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QPushButton" name="pbDibujar">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Dibujar una Im√°gen del Disco</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pbBuscar">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextEdit" name="teHTML">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leURL">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lURL">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lGuardado">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Donde Guardar:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="leGuardado"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
